src/ray/gcs/gcs_server/gcs_worker_manager.cc

```diff
@@ -21,44 +21,50 @@ void GcsWorkerManager::HandleReportWorkerFailure(
     const rpc::ReportWorkerFailureRequest &request, rpc::ReportWorkerFailureReply *reply,
     rpc::SendReplyCallback send_reply_callback) {
   const rpc::Address worker_address = request.worker_failure().worker_address();
-  RAY_LOG(DEBUG) << "Reporting worker failure, " << worker_address.DebugString();
+  const auto worker_id = WorkerID::FromBinary(worker_address.worker_id());
+  const auto node_id = ClientID::FromBinary(worker_address.raylet_id());
+  RAY_LOG(WARNING) << "Reporting worker failure, worker id = " << worker_id
```

Comment:
Can we also make it INFO if `intentional_disconnect` is true?

Reply:
good idea.